cmd/create/cluster/cmd.go

```diff
@@ -2209,6 +2219,47 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	// Audit Log Forwarding
+	auditLogRoleARN := args.AuditLogRoleARN
+
+	if auditLogRoleARN != "" && !isHostedCP {
+		r.Reporter.Errorf("Audit log forwarding to AWS CloudWatch is only supported for Hosted Control Plane clusters")
+		os.Exit(1)
+	}
+
+	if interactive.Enabled() && isHostedCP {
+		requestAuditLogForwarding, err := interactive.GetBool(interactive.Input{
+			Question: "Enable audit log forwarding to AWS CloudWatch (optional)",
+			Default:  false,
+			Required: true,
```

Comment:
Is this required (in the question is marked as optional).

Reply:
The answer (yes or no) is required for the question prompt.
You are correct that the '(optional)' does not align with the user requirement that they must input a yes or no answer.
Will go ahead and remove the '(optional)' string.